src/pages/AppDetail.svelte

```diff
@@ -49,7 +50,11 @@
       <td><code>{app.app_id}</code></td>
     </tr>
   </table>
-
+  {#if app.prototype}
+    <AppAlert
+      message="The glean-js application is a prototype. The metrics and pings listed below may contain inconsistencies and testing strings."
```

Comment:
Oops, I forgot to notice on my last review that in this message, we should dinamically get the application name.